.github/workflows/update-opengrep.yml

```diff
@@ -0,0 +1,67 @@
+name: Update Opengrep
+
+on:
+  schedule:
+    # Run every Monday at 10:00 AM UTC
+    - cron: '0 10 * * 1'
+  workflow_dispatch: # Allow manual triggering
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-opengrep:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
+
+      - name: Setup Python
```

Comment:
we should be using vpython in depot tools

Reply:
fixed in https://github.com/brave/brave-core/pull/32195/commits/0ac7bb6a8ee8285f954ace9dc8d7aa83207fb42a